src/transformers/modeling_dpr.py

```diff
@@ -61,28 +161,31 @@ def forward(
         inputs_embeds: Optional[Tensor] = None,
         output_attentions: bool = False,
         output_hidden_states: bool = False,
-    ) -> Tuple[Tensor, ...]:
+        return_tuple: bool = False,
+    ) -> Union[BaseModelOutputWithPooling, Tuple[Tensor, ...]]:
         outputs = self.bert_model(
             input_ids=input_ids,
             attention_mask=attention_mask,
             token_type_ids=token_type_ids,
             inputs_embeds=inputs_embeds,
-            output_hidden_states=True,
```

Comment:
Don't know why this was always `True`, but it ended up messing with the tests with the change of output. Was this on purpose @lhoestq ?

Reply:
You can remove it. I was using it before but not anymore